ports.mk

```diff
@@ -556,7 +556,7 @@ iphone: $(DETECT_OBJS) $(OBJS)
 	+$(LD) $(LDFLAGS) -o scummvm $(DETECT_OBJS) $(OBJS) \
 		$(OSX_STATIC_LIBS) \
 		-framework UIKit -framework CoreGraphics -framework OpenGLES -framework GameController \
-		-framework CoreFoundation -framework QuartzCore -framework Foundation \
+		-framework CoreFoundation -framework QuartzCore -framework Foundation -framework Accelerate \
```

Comment:
I guess we also need to add it in `configure`:
https://github.com/scummvm/scummvm/blob/7e3d5513217b450a2518a04f2ede25c2b2d190f1/configure#L3885-L3889